src/main/kotlin/com/team4099/robot2023/config/ControlBoard.kt

```diff
@@ -21,10 +21,10 @@ object ControlBoard {
     }
 
   val strafe: Double
-    get() = -driver.leftXAxis
+    get() = driver.leftXAxis
 
   val forward: Double
-    get() = -driver.leftYAxis
+    get() = driver.leftYAxis
 
```

Comment:
```
git rebase -i HEAD~5 # or however many commits back you need to go
# change pick to edit for this commit
# exit the editor, you will pause on this commit
git reset --soft HEAD~1 # this will undo the commit you selected to edit but leave your changes present in the index
# stage the changes you actually want in this commit, then commit them (not amend, your edited commit is gone now)
# stage the remaining changes and commit them in a separate commit
git rebase --continue
```